scripts/Docker/readme.md

```diff
@@ -0,0 +1,76 @@
+# Librealsense Docker
+
+## Pre-Work: Docker Installation
+Install docker in the environemnt using the  following tutorial (adjust the OS):
+https://www.digitalocean.com/community/tutorials/how-to-install-and-use-docker-on-ubuntu-16-04
+
+## Dockerfile description
+- Ubuntu base system (Ubuntu 20.04 by default)
+- **librealsense-builder** stage - builds the binaries and has all the build dependencies 
+- **librealsense** stage -  contain only the built binaries and the required runtime dependencies (~60MB)
+- Support for Python bindings (python not included) and networking
+- Binaries are stripped during build stage to minimize image size
+- Support scripts for building and running the image are also included
+Add python version, openGL?, self info to be printed
+
+# Getting librealsense docker - pre-built
+
+As long as the repo is private, login is needed:
+1. add the password in some password.txt file
+2. run: 
+```
+cat password.txt | docker login –username=“…” –password-stdin
```

Comment:
Need to think of either dropping PW, or creating alternative path

Reply:
Done